src/app.module.ts

```diff
@@ -3,8 +3,22 @@ import { UserModule } from './user/user.module';
 import { AuthModule } from './auth/auth.module';
 import { RecipeModule } from './recipe/recipe.module';
 import { ConfigModule } from '@nestjs/config';
+import { CacheModule } from '@nestjs/cache-manager';
+import * as redisStore from 'cache-manager-redis-store';
 
 @Module({
-  imports: [UserModule, AuthModule, RecipeModule, ConfigModule.forRoot()],
+  imports: [
+    UserModule,
+    AuthModule,
+    RecipeModule,
+    ConfigModule.forRoot(),
+    CacheModule.register({
+      isGlobal: true,
+      store: redisStore,
+      host: process.env.REDIS_HOST,
+      port: process.env.REDIS_PORT,
+      ttl: parseInt(process.env.TTL_IN_SECONDS),
```

Comment:
I would always provide radix, 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt
"this does not always default to 10!"